chain/chain/src/flat_storage_resharder.rs

```diff
@@ -1025,9 +1025,10 @@ fn copy_kv_to_child(
 
     // Sanity check we are truly writing to one of the expected children shards.
     if new_shard_uid != *left_child_shard && new_shard_uid != *right_child_shard {
-        let err_msg = "account id doesn't map to any child shard!";
-        error!(target: "resharding", ?new_shard_uid, ?left_child_shard, ?right_child_shard, ?shard_layout, ?account_id, err_msg);
-        return Err(Error::ReshardingError(err_msg.to_string()));
+        let err_msg = "account id doesn't map to any child shard! - skipping it";
+        warn!(target: "resharding", ?new_shard_uid, ?left_child_shard, ?right_child_shard, ?shard_layout, ?account_id, err_msg);
```

Comment:
Maybe let's use `debug_assert`, ideally something that would not silently pass in forknet testing.

Reply:
The purpose of this PR is to unblock forknet testing though :(

Reply:
I agree with both, can you add a todo to change it to debug_assert once we fix the root cause?